public/components/Sidebar/Sidebar.js

```diff
@@ -11,10 +11,28 @@ export default class Sidebar extends React.Component {
     )
   };
 
+  onTerritoryChange = (e) => {
+    let territory = e.target.value;
+    this.props.uiActions.setTerritory(territory);
+  }
+
   render () {
     return (
       <div className="sidebar">
         <div className="sidebar__link-group">
+        <div className="sidebar__link-group__header">Select a territory:</div>
+
+          <form className="pure-form pure-form-stacked">
+            <div className="pure-u-1 pure-u-md-1-5" style={{padding: '10px'}}>
+              <select id="territory" style={{width: '100%'}} onChange={(e) => this.onTerritoryChange(e)} value={this.props.territory}>
```

Comment:
stylesheet for these? hard to find in the future.

Reply:
I don't like CSS 😆